src/routes/safe/components/Apps/hooks/useIframeMessageHandler.ts

```diff
@@ -104,6 +101,37 @@ const useIframeMessageHandler = (
           break
         }
 
+        case SDK_MESSAGES.RPC_CALL: {
+          const payload = messagePayload as SDKMessageToPayload['RPC_CALL']
+
+          if (
+            web3ReadOnly &&
+            web3ReadOnly.currentProvider != null &&
```

Comment:
missing `=`

Reply:
Actually not, this way it checks also for undefined

Reply:
but `currentProvider` is either `null` or a string or the provider object

Reply:
I don't get why you push this change, because it would still check for null, but ok.